cypress/integration/seededFlows/toggleFeatureFlags.spec.js

```diff
@@ -0,0 +1,10 @@
+describe('Toggling feature flags', () => {
+  it('toggles and verifies featue flags', () => {
```

Comment:
```suggestion
  it('toggles and verifies feature flags', () => {
```

I feel like the ideal test would be to disable/enable a feature flag, then try to visit that route and verify the in-browser behaviour.... but I understand why for now this might be the best approach until we start building out cypress tests for feature-flagged areas 👍

Reply:
My thinking was as follows:

1. A feature flag by itself doesn't have any routes or anything. So I would need to scaffold a route, controller and view at least, which seems like overkill for testing such a simple controller.
2. I could have used an existing feature flag but the problem is that that's likely to be removed at some point and I wanted a test that can remain unchanged independent of a specific flag.